src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -6319,5 +6319,66 @@ .maxstack  3
   IL_009e:  ret
 }");
         }
+
+        [Fact, WorkItem(47191, "https://github.com/dotnet/roslyn/issues/47191")]
+        public void AssignStaticStructField()
```

Comment:
Please add a test where we grab a `static` field w/o a qualifier by means of `using static`. Basically not just a direct `static` field access on the current type.